contracts/LifToken.sol

```diff
@@ -17,20 +24,14 @@ contract LifToken is MintableToken, Pausable {
   event TransferData(address indexed from, address indexed to, uint value, bytes data);
   event ApprovalData(address indexed from, address indexed spender, uint value, bytes data);
 
-  function transfer(address _to, uint256 _value) whenNotPaused returns (bool) {
-    return super.transfer(_to, _value);
-  }
-
-  function approve(address _spender, uint256 _value) whenNotPaused returns (bool) {
-    return super.approve(_spender, _value);
-  }
-
-  function transferFrom(address _from, address _to, uint256 _value) whenNotPaused returns (bool) {
-    return super.transferFrom(_from, _to, _value);
-  }
-
-  // approveData is an addition to ERC20 token methods. It allows approving the transference of value and execute a data call on the approval transaction
-  function approveData(address spender, uint value, bytes data) whenNotPaused {
+  /**
+   * @dev `approveData` is an addition to ERC20 token methods.
+   * It allows to approve the transfer of value and execute a call with data in the same transaction
```

Comment:
It allows to approve the transfer of value and execute a call with the data sent